spinta/core/enums.py

```diff
@@ -39,3 +44,91 @@ class Level(enum.IntEnum):
 
     # Data is linked with a known vocabulary.
     linked = 5
+
+
+class Status(enum.IntEnum):
+    """Status of the data record"""
+
+    # Status isn't given
+    absent = 0
+
+    # The data of this element is being updated. This field can be changed without any warning
+    develop = 1
+
+    # Data in this element is completed. The data of this row can be changed with a warning issued
+    # before time X, where X is not less than 12 months
+    completed = 2
+
+    # Data isn't being updated, but the element isn't planned for removing
+    discont = 3
+
+    # For elements which are planned for removal.
+    deprecated = 4
+
+    # Withdrawn and isn't used anymore. Can be set after the field has been in the `deprecated` status for X time.
+    withdrawn = 5
+
+
+class Visibility(enum.IntEnum):
+    """Visibility of the data, which means access to this metadata."""
+
+    # No visibility is given
+    absent = 0
+
+    # Metadata isn't published.
+    private = 1
+
+    # Application on the level of the informational system.
+    protected = 2
+
+    # Application on the level of the country.
+    package = 3
+
+    # Application on the level of the EU.
+    public = 4
+
+
+def load_level(
+    component: 'spinta.components.Component',
+    given_level: Level | int | str
+):
+    if given_level:
+        if isinstance(given_level, Level):
+            level = given_level
+        else:
+            if isinstance(given_level, str) and given_level.isdigit():
+                given_level = int(given_level)
+            if not isinstance(given_level, int):
+                raise InvalidLevel(component, level=given_level)
+            level = enum_by_value(component, 'level', Level, given_level)
+    else:
+        level = None
+    component.level = level
+
+
+#  enum loaders
+
+def load_status(
+    component: 'spinta.components.Component',
+    given_status: Status | str
+):
+    if not isinstance(given_status, Status):
+        status = given_status or component.status
+        status = enum_by_name(component, 'status', Status, status)
+    else:
+        status = given_status
+    component.status = status
+    component.given.status = given_status
```

Comment:
Pasiūlymas, kadangi tiek `if` tiek `else` dalys turi tą pačią kodo dalį:
```suggestion
    status = given_status
    
    if not isinstance(given_status, Status):
        status = enum_by_name(component, 'status', Status, status or component.status)
        
    component.status = status
    component.given.status = given_status
```